src/app/components/tabs/tabs-demo.component.html

```diff
@@ -11,3 +11,15 @@
     This tab cannot be closed.
   </sky-tab>
 </sky-tabset>
+
+<h3>Tabs example with counter</h3>
```

Comment:
This may be fine but it wouldn't surprise me if they don't just want you to add another tab to the previous example that has a header count. I've seen similar with other components.